test/HttpApiHandlerTests.cs

```diff
@@ -269,6 +271,46 @@ public void CreateCheckStatus_Returns_Corrent_HttpManagementPayload_based_on_cus
             var response = JsonConvert.DeserializeObject<JObject>(content);
             Assert.Equal(response["runtimeStatus"], runtimeStatus.ToString());
         }
+        [Fact]
+        public async Task GetAllStatus_is_Success()
+        {
+
+            var list = (IList<DurableOrchestrationStatus>)new List<DurableOrchestrationStatus>
+                     {
+                         new DurableOrchestrationStatus
+                         {
+                             InstanceId = "01",
+                             RuntimeStatus = OrchestrationRuntimeStatus.Running
+                         },
+                         new DurableOrchestrationStatus
+                         {
+                             InstanceId = "02",
+                             RuntimeStatus = OrchestrationRuntimeStatus.Completed
+                         },
+                     };
+
+            var clientMock = new Mock<DurableOrchestrationClientBase>();
+            clientMock
+                .Setup(x => x.GetStatusAsync(It.IsAny<CancellationToken>()))
+                .Returns(Task.FromResult(list));
+            var httpApiHandler = new ExtendedHttpApiHandler(clientMock.Object);
+
+            var getStatusRequestUriBuilder = new UriBuilder(TestConstants.NotificationUrl);
+            getStatusRequestUriBuilder.Path += $"/Instances/";
+
+            var responseMessage = await httpApiHandler.HandleRequestAsync(
+                new HttpRequestMessage
+                {
+                    Method = HttpMethod.Get,
+                    RequestUri = getStatusRequestUriBuilder.Uri,
+                });
+            var actual = JsonConvert.DeserializeObject<IList<StatusResponsePayload>>(await responseMessage.Content.ReadAsStringAsync());

```

Comment:
Before this, I suggest you add a check for HTTP 200 status code.

Reply:
Done!